tools/fonts/config/valid_codes.js

```diff
@@ -52,21 +52,51 @@ module.exports = {
   noteheadWhole: 'v1d',
   noteheadHalf: 'v81',
   noteheadBlack: 'vb',
+  noteheadXDoubleWhole: 'vxx',
   noteheadXWhole: 'v92',
   noteheadXHalf: 'v95',
   noteheadXBlack: 'v3e',
-  noteheadCircleX: 'v3b',
+  noteheadCircleXDoubleWhole: 'vxx',
+  noteheadCircleXWhole: 'vxx',
+  noteheadCircleXHalf: 'vxx',
+  noteheadCircleX: 'vxx',
+  noteheadCircledDoubleWhole: 'vxx',
+  noteheadCircledWhole: 'vxx',
+  noteheadCircledHalf: 'vxx',
+  noteheadCircledBlack: 'vxx',
+  noteheadDoubleWholeSquare: 'vxx',
+  noteheadSquareWhite: 'vxx',
+  noteheadSquareWhite: 'vxx',
+  noteheadSquareBlack: 'vxx',
+  noteheadTriangleDownDoubleWhole: 'vxx',
+  noteheadTriangleDownWhole: 'vxx',
+  noteheadTriangleDownHalf: 'vxx',
+  noteheadTriangleDownBlack: 'vxx',
+  noteheadTriangleUpDoubleWhole: 'vxx',
   noteheadTriangleUpWhole: 'v49',
   noteheadTriangleUpHalf: 'v93',
   noteheadTriangleUpBlack: 'v40',
-  noteheadDiamondWhole: 'v46',
```

Comment:
Just a quick question here. This changed from v46 to v27. Since I'm not familiar with these v codes, I'm just noting this to make sure it was an intentional change. :-)

Reply:
yes it was, the code was not correctly mapped. You can look at the codes with

tools/fonts/inspector/gonville.html